Project.toml

```diff
@@ -30,7 +30,7 @@ ApproxFunOrthogonalPolynomialsPolynomialsExt = "Polynomials"
 ApproxFunOrthogonalPolynomialsStaticExt = "Static"
 
 [compat]
-ApproxFunBase = "0.8.57, 0.9.13"
+ApproxFunBase = "0.9.15"
```

Comment:
Could you update this to "0.8.58, 0.9.15"? The change should be backported after https://github.com/JuliaApproximation/ApproxFunBase.jl/pull/597

Reply:
Done 👍